src/pythonapi/DevsimDoc.cc

```diff
@@ -1027,7 +1027,7 @@ R"(    devsim.load_devices (file)
 )";
 
 static const char write_devices_doc[] =
-R"(    devsim.write_devices (file, device, type)
+R"(    devsim.write_devices (file, device, type, include)
```

Comment:
The docstrings are actually generated in a different project.
[https://github.com/devsim/devsim_documentation/blob/main/pydoc/meshingCommands.py#L222](https://github.com/devsim/devsim_documentation/blob/main/pydoc/meshingCommands.py#L222)
However I am happy to update this based  on what you have here.